lighthouse-plugin-ad-speed-insights/audits/ad-request-critical-path.js

```diff
@@ -128,7 +156,21 @@ function computeDepth(requests) {
  * @param {string} url
  * @return {string}
  */
-function requestName(url) {
+function getAbbreviatedUrl(url) {
+  const u = new URL(trimQuery(url));
+  const parts = u.pathname.split('/');
+  if (parts.length > 4) {
```

Comment:
Why not just set a max url length instead? I'm sure there are cases with very long parts, containing params, etc.

Reply:
This makes more sense for comparing two requests IMO. Lighthouse will shorten long strings so that's not something we need to do here.

Reply:
Capped it at 60